### DISCUSSION TITLE
Flicker/Tearing

### DISCUSSION THREAD

**Discussion by Pramod-Viswanath** [ocornut/imgui#5099] [New Users Build/Link/Run issues ONLY!]
HIi,

I am new to imgui. I am exploring the library with glfw3. Running the demo application on Dell inspiron workstation laptop that has NVIDIA graphics card. When I run the application and drag the demo window, I get tearing and flicker.

What am I missing?

Imgui v1.88 WIP / docking branch, built locally and referred the headers.

Regards,
Pramod.
![Tearing](https://user-images.githubusercontent.com/79860275/158001022-7c16b0fd-ea7d-412a-9177-af044f23c9f5.PNG)

**Comment by PathogenDavid** [comment]
To clarify: Are you one of the examples unmodified or your own app? Also which rendering backend are you using?

It looks like the backbuffer isn't getting cleared.

**Comment by Pramod-Viswanath** [comment]
I haven't modified any code of examples. Just using it as is. I am using imgui_impl_opengl3.

**Comment by Pramod-Viswanath** [comment]
David, thanks for the hint about back buffer. I was calling glfwSwapBuffers(window); twice in the loop which caused intense flicker nad tearing. Thanks!
